app/Http/Controllers/Flights/FlightController.php

```diff
@@ -77,15 +81,16 @@ public function store(Request $request)
 
 		$users = UserNotification::whereJsonContains('new_request->airports', $request->departure)
 								->orWhereJsonContains('new_request->airports', $request->arrival)
-								->orWhereJsonContains('new_request->aircrafts', $request->airport)
+                                ->orWhereJsonContains('new_request->aircrafts', $request->airport)
+                                ->where('user_id', '!=', Auth::id())
 								->with('user')
 								->get()
 								->pluck('user')
 								->flatten();
```

Comment:
Is it possible to tidy this up into a method in the `UserNotification` class?

Reply:
For now, this will work, can be merged into the `UserNotification` class later